modP2p/src/org/aion/p2p/V1Constants.java

```diff
@@ -16,6 +16,10 @@
     /** Limits the number of key-value pairs returned to one trie data request. */
     public static final int TRIE_DATA_REQUEST_MAXIMUM_BATCH_SIZE = 100;
 
+    /** Limits the number of blocks returned to one blocks request. */
+    // TODO: also add limit for the size of the resulting message
+    public static final int BLOCKS_REQUEST_MAXIMUM_BATCH_SIZE = 60;
```

Comment:
in the current p2pmgr features. The message does not be chunk into small pieces. Therefore need to consider the transfer within 1secs under the full energy consumed blocks condition.

Reply:
Yes, I realize that I have not yet considered limitations regarding the message size. That's why I left the TODO above for future enhancement.